docs/content/releases.md

```diff
@@ -10,6 +10,7 @@ docs: "DOCS-616"
 29 Mar 2023
 
 OVERVIEW:
+* **IMPORTANT** Beginning with release 3.1.0 the NET_BIND_SERVICE capability is no longer used, and instead relies on net.ipv4.ip_unprivileged_port_start sysctl to allow port binding. Kubernetes 1.22 or later is required for this sysctl to be [classified as safe](https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#safe-and-unsafe-sysctls). **Ensure** that you are using the latest updated `deployment` and `daemonset` examle yaml files available in the repo.
 * *The minimum supported version of Kubernetes is now 1.22*. The NGINX Ingress Controller now uses `sysctls` to [bind to lower level ports without privilege escalation](https://github.com/nginxinc/kubernetes-ingress/pull/3573/). This removes the need to use `NET_BIND_SERVICE` to bind to these ports. Thanks to [Valters Jansons](https://github.com/sigv) for making this feature possible!
 * Added support for loading pre-compiled [AppProtect Policy Bundles](https://github.com/nginxinc/kubernetes-ingress/pull/3560) when using the `-enable-app-protect` cli argument. This feature removes the need for the Ingress Controller to re-compile App Protect when NGINX reloads, and will instead use a pre-compile policy bundle. See [App Protect WAF Bundles](https://docs.nginx.com/nginx-ingress-controller/app-protect-waf/configuration/#app-protect-waf-bundles) for examples and configuration details.
 * IngressMTLS policy now supports configuring a Certificate Revocation Lists(CRL). When using this feature requests made using a revoked certificate will be rejected. See [Using a Certificate Revocation List](https://docs.nginx.com/nginx-ingress-controller/configuration/policy-resource/#using-a-certificate-revocation-list) for details on configuring this option.
```

Comment:
"When using this feature requests made using a revoked certificate will be rejected." We should not need to describe what the impact of a revoked cert in a CRL means.
Try this:
"This enhancement allows the CRL to be presented as either a Kubernetes secret (limited to 1MB) or as a CRL file to the IngressMTLS Policy"

(I was guessing on the 1MB, I can't recall at the moment but it is the size limit of a K8s secret I was reaching for)